spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py

```diff
@@ -320,6 +320,13 @@ def name_to_type(self, name):
     """Determine Custom Descriptor type name for the given metric type name."""
     return self.__metric_name_prefix + name
 
+  def distribution_to_counter(self, name):
+    """Converts a distribution name (or type) into a corresponding counter.
+    
+    The counter is used for convienence in determine the count in the distribution.
+    """
+    return name + '_count'
```

Comment:
Previously `count` was prefixed with two underscores: `__count`. Is this change intentional?

Reply:
yea, that is the "stackdriver compliant naming policies" I mentioned. It is lexically valid, but the stackdriver folks frowned at the convention.

I only do this for the distribution counts, which is implicitly using the standard stackdriver policies configuration (which requires distributions instead of the individual dimension counters, which are still decorated with double-_ to ensure name disambiguation)

Reply:
:+1: